lib/saml_idp/xml_security.rb

```diff
@@ -120,7 +121,9 @@ def request?
 
       # matches RubySaml::Utils
       def build_query(params)
-        type, data, relay_state, sig_alg = [:type, :data, :relay_state, :sig_alg].map { |k| params[k]}
+        type, data, relay_state, sig_alg = %i[type data relay_state sig_alg].map do |k|
+          params[k]
+        end
```

Comment:
maybe for a future PR since this is technically not a syntax change, but we could simplify to this
```suggestion
        type, data, relay_state, sig_alg = params.values_at(:type, :data, :relay_state, :sig_alg)
```

Reply:
well, we're already here!